cmd/rook/block/delete.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package block
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/model"
+	"github.com/rook/rook/pkg/rook/client"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var (
+	deleteImageName     string
+	deleteImagePoolName string
+)
+
+var deleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Deletes a block image from the cluster",
+}
+
+func init() {
+	deleteCmd.Flags().StringVar(&deleteImageName, "name", "", "Name of block image to delete (required)")
+	deleteCmd.Flags().StringVar(&deleteImagePoolName, "pool-name", "rbd", "Name of storage pool to delete block image from")
```

Comment:
Ceph automatically creates the `rbd` pool by default and it's what the `block create` uses by default too.  We should keep the default pool for `block delete` the same as `block create`.

Also since `rook` runs in a standalone mode outside of kubernetes, i think rbd pool is a good default here.